common/hardcoded_params/package.xml

```diff
@@ -18,6 +18,13 @@
   <maintainer email="carma@dot.gov">carma</maintainer>
   <license>Apache 2</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION != 2">catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_VERSION == 2">ament_cmake</buildtool_depend>
+  
+  <build_depend condition="$ROS_VERSION == 2">ament_cmake_auto</build_depend>
   <build_depend>carma_cmake_common</build_depend>
+
+  <export>
+    <build_type condition="$ROS_VERSION == 2">ament_cmake</build_type>
```

Comment:
Shouldn't there be a ros1 build type export condition as well?

Reply:
I had originally included it, but actually received some errors from its presence. Since catkin packages don't require it, my preference is to not add it until there is a need.